contracts/sol6/KyberNetwork.sol

```diff
@@ -1105,24 +1126,28 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         uint256 totalBps;
         uint256 srcDecimals = (src == ETH_TOKEN_ADDRESS) ? ETH_DECIMALS : reservesData.decimals;
         uint256 destDecimals = (src == ETH_TOKEN_ADDRESS) ? reservesData.decimals : ETH_DECIMALS;
-        
+
         for (uint256 i = 0; i < reservesData.addresses.length; i++) {
             if (i > 0 && (uint256(reservesData.ids[i]) <= uint256(reservesData.ids[i - 1]))) {
                 return 0; // ids are not in increasing order
             }
+            //+ totalBps <= BPS (end of this function)
             totalBps += reservesData.splitsBps[i];
 
+            //+ destAmount <= MAX_QTY * MAX_RATE == 1e53
             destAmount += calcDstQty(
                 reservesData.srcAmounts[i],
                 srcDecimals,
                 destDecimals,
                 reservesData.rates[i]
             );
 
+            //+ feeAccountedBps <= BPS
```

Comment:
Should add comment that this is only one trade, so `feeAccountedBps <= BPS`, since other places we comment that `feeAccountedBps <= 2 * BPS`

Reply:
Oh yes. right.